src/libsync/creds/keychainchunk.h

```diff
@@ -159,6 +175,39 @@ private slots:
 #endif
 }; // class ReadJob
 
+/*
+* @brief: Simple wrapper class for QKeychain::DeletePasswordJob
+*/
+class OWNCLOUDSYNC_EXPORT DeleteJob : public KeychainChunk::Job
+{
+    Q_OBJECT
+public:
+    DeleteJob(Account *account, const QString &key, const bool &keychainMigration, QObject *parent = nullptr);
```

Comment:
Just `bool` and not `const bool &`

Reply:
Ah, you should have a look at `src/gui/tray/UserModel.cpp` too then in another PR, I've seen this there too.

Reply:
Yeah clearly might have missed it at other places.